src/Concordium/Client/Commands.hs

```diff
@@ -850,6 +857,20 @@ transactionPLTTransferCmd =
             (progDesc "Transfer tokens from one account to another.")
         )
 
+transactionPLTUpdateSupplyCmd :: Mod CommandFields TransactionCmd
+transactionPLTUpdateSupplyCmd =
+    command
+        "update-supply-plt"
+        ( info
+            ( TransactionPLTUpdateSupply
+                <$> strOption (long "action" <> metavar "ACTION" <> help "The action (either `mint` or `burn`)")
+                <*> option (eitherReader tokenAmountFromStringInform) (long "amount" <> metavar "TOKEN-AMOUNT" <> help "Amount of tokens to send.")
+                <*> strOption (long "tokenId" <> metavar "TOKEN_ID" <> help "Token id (Symbol) of the token.")
+                <*> transactionOptsParser
+            )
+            (progDesc "Mint or burn plt tokens.")
+        )
```

Comment:
I think `concordium-client transaction update-supply-plt mint` is not a great user interface - it's long and difficult to remember. I'd suggest actually we gather all the PLT-related commands under a top level `plt` and have:
- `plt send` (currently `transaction transfer-plt`; `send` is congruent with the `transaction send` operation for sending CCDs).
- `plt mint` (rather than `transaction update-supply-plt mint`)
- `plt burn` (rather than `transaction update-supply-plt burn`)